populous/bloom.py

```diff
@@ -1,4 +1,4 @@
-from peloton_bloomfilters import BloomFilter as _BloomFilter
+from pybloom_live import BloomFilter as _BloomFilter
```

Comment:
Actually this lib is very slow. The better lib I've found for Python 3 is `pybloomfilter3`.

The best solution would be to use either `peloton_bloomfilters` or `pybloomfilter3` according to which Python version we are using.